src/v/pandaproxy/schema_registry/types.h

```diff
@@ -534,6 +535,11 @@ from_string_view<compatibility_level>(std::string_view sv) {
       .default_match(std::nullopt);
 }
 
+struct compatibility_result {
+    bool is_compat;
+    std::vector<ss::sstring> messages;
```

Comment:
> q: in practice I don't think it would make much difference, but should we use `chunked_vector` here as a matter of course?

It would require 8192 errors before becoming a problem.

Reply:
yeah, just a knee-jerk response to `std::vector` at this point